### DISCUSSION TITLE
Lazer black bar resolution scaling for Ultrawide users broken/gone?

### DISCUSSION THREAD

**Discussion by FrontlineLucian** [ppy/osu#21457] [Q&A] [answered]
### Type

Game behaviour

### Bug description

A recent Update seems to have destroyed scaling to different resolutions? I play on a 3440 x 1440 Ultrawide and instead of giving me a 16:9 game with black bars as expected in Fullscreen mode, the game now just steps down in resolution and then stretches over the entire monitor making it unplayable for me. The game also seems to just decrease my monitors resolution instead of the resolution the game is rendered in? Not sure about that one though (looks like that when i tab out of the game)

### Screenshots or videos

![PXL_20221129_205703463](https://user-images.githubusercontent.com/64693922/204646443-547e9279-a5d3-4fc0-b0f9-f138c1ea7214.jpg)

### Version

2022.1123.0

### Logs

[network.log](https://github.com/ppy/osu/files/10117253/network.log)
[performance.log](https://github.com/ppy/osu/files/10117254/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10117255/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10117256/updater.log)
[database.log](https://github.com/ppy/osu/files/10117257/database.log)
[input.log](https://github.com/ppy/osu/files/10117258/input.log)

**Comment by Susko3** [comment, marked as answer]
When using a 16:9 resolution on a 21:9, the appearance of the black bars/stretching is controlled by
a) your GPU, if GPU scaling is enabled, and if it's not
b) your monitor

If you wish to use 16:9 with black bars, it's recommended to keep osu! and your monitor at native resolution and use
`Screen scaling: Everything` and adjusting the vertical and horizontal scale as appropriate.

![image](https://user-images.githubusercontent.com/16479013/204647709-69a48705-5939-4f61-80fb-a8238664e4bf.png)
